src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -37,7 +37,11 @@ static void OnIsRefreshingPropertyChanged(BindableObject bindable, object oldVal
 
 			var refreshView = (RefreshView)bindable;
 			refreshView.Refreshing?.Invoke(bindable, EventArgs.Empty);
-			refreshView.Command?.Execute(refreshView.CommandParameter);
+
+			if (refreshView.Command is not null && refreshView.Command.CanExecute(refreshView.CommandParameter))
```

Comment:
`if (refreshView.Command?.CanExecute(refreshView.CommandParameter) is true)`
FYI I'm not a reviewer but this is cleaner